src/main/java/meilisearch/MeilisearchIndex.java

```diff
@@ -0,0 +1,140 @@
+package meilisearch;
+
+import java.io.Serializable;
+
+import com.google.gson.*;
+import lombok.Getter;
+import lombok.ToString;
+
+/**
+ * Meilisearch index
+ */
+@ToString
+public class MeilisearchIndex implements Serializable {
```

Comment:
This class makes an extended use of the Lombok library to limit boilerplate code that was cluttering the class.

It enables us to not having to create a custom toString method as well as default getters and setters.